civihr_default_theme/scss/pages/_hr-details.scss

```diff
@@ -65,3 +72,192 @@
     }
   }
 }
+
+.view-mydetails-personal {
+  .chr_panel--my-details__personal__contact_id span {
+    @include views-label;
+  }
+
+  .pane-block {
+    border-bottom: 1px solid $gray-light;
+    margin-bottom: 0;
+  }
+
+  .views-row {
+    min-height: 80px;
+    padding-left: 0;
+    position: relative;
+    @media (min-width: 320px) {
+      padding-left: 100px;
+    }
+  }
+
+  .views-field-image-URL {
+    display: none;
+    left: 0;
+    position: absolute;
+    top: 0;
+    width: 80px;
+    @media (min-width: 320px) {
+      display: block;
+    }
+  }
+
+  .chr_panel__footer {
+    border-top: 0;
+    margin-top: 0;
+  }
+
+  .views-field-nothing {
+    .field-content {
+      display: flex;
+      flex-wrap: wrap;
+      justify-content: space-between;
+      white-space: nowrap;
+      width: 100%;
+    }
+
+    .chr_panel--my-details__personal__contact_id,
+    .chr_panel--my-details__personal__gender,
+    .chr_panel--my-details__personal__marital_status {
+      margin-right: 10px;
+    }
+  }
+
+  .views-field-ni-ssn-70 .field-content {
+    width: 100%;
+  }
+
+  .chr_panel--my-details__personal__application_status--in-progress {
+    display: flex;
+
+    > * {
+      width: 50%;
+    }
+  }
+
+  .views-field.views-field-display-name {
+    color: $crm-gray-darker-alt;
+    font-size: $crm-font-size-large;
+    margin-bottom: 5px;
+
+    .views-label {
+      display: none;
+    }
+
+    > * {
+      width: 100%;
+    }
+  }
+
+  .chr_panel--my-details__personal__application_status {
+    > * {
+      display: inline-block;
+      float: left;
+      width: 50%;
+    }
+
+    &.not-applying,
+    &.No {
```

Comment:
Is this class auto generated?

Reply:
Nope. BEM? Could you wrote how it would be with BEM? Sometimes I get confused. Thanks.

Reply:
@deb1990 ^

Reply:
What happened here? The plan was `if not required remove it. If required please move the logic to PHP`.

Reply:
Removed in next update. You should update the other related PR to not see any errors because I had changed templates in views.